src/styles/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+  &__info {
+    margin-top: 40px;
+  }
+
+  &__info-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__info-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin: 4px 0 16px;
+  }
+
+  &__info-reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 23px 0 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: rgba(97, 96, 112, 1);
+    line-height: 18px;
+
+  }
+
+  &__price-num {
+    font-size: 16px;
+    font-weight: 700;
+
+  }
+
+  &__buy {
+    text-decoration: none;
+    color: #fff;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    background-color: #00acdc;
+    padding: 12px 70px;
+    border-radius: 5px;
+  }
+
+  &__buy:hover {
+    background-color: #fff;
+    color: #00acdc;
+    border: 1px solid #00acdc;
+    padding: 11px 69px;
+    transition: 0.4s ease-out;
```

Comment:
Transition should be added for the default state of the element, not hover, so it works smoothly both ways.
Also, better specify the property you want to apply a transition to
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x